### PR TITLE
feat: fixed sidebar and navbar (#635)

### LINKED ISSUE #635: Fixed position for navbar/sidebar
Navbar / sidebar fixed, content scrollable

### PR DESCRIPTION
**Pull request by smellyshovel**
fixes #635

Introduce fixed sidebar and navbar (admin-wide, related components are left untouched)
Please, make sure to **check out the related issue** before merging

### CODE REVIEW

File: src/components/admin/app-navbar/AppNavbar.vue [hunk 1]
```diff
@@ -94,6 +94,11 @@ export default {
 </script>
 <style lang="scss">
 .app-navbar {
+  position: fixed !important;
```
Comment (asvae) [inline src/components/admin/app-navbar/AppNavbar.vue]: Maybe makes more sense to move this to vuestic-ui.

File: src/components/admin/app-sidebar/AppSidebar.vue [hunk 2]
```diff
@@ -53,3 +53,10 @@ export default {
 }
 
 </script>
+
+<style lang="scss">
+  .app-sidebar {
```
Comment (asvae) [inline src/components/admin/app-sidebar/AppSidebar.vue]: Same here.

File: src/components/admin/app-navbar/AppNavbar.vue (hunk 1 above)
Comment (Kreezag) [reply src/components/admin/app-navbar/AppNavbar.vue]: fixed

File: src/components/admin/app-sidebar/AppSidebar.vue (hunk 2 above)
Comment (Kreezag) [reply src/components/admin/app-sidebar/AppSidebar.vue]: fixed

**Pull request merged**